src/visualizers/panels/PlotlyGraph/PlotlyGraphControl.js

```diff
@@ -0,0 +1,169 @@
+/*globals define, WebGMEGlobal*/
+
+define([
+    'js/Constants'
+], function (
+    CONSTANTS
+) {
+    'use strict';
+
+    const GRAPH = 'Graph',
+        DATA = 'data';

```

Comment:
I would probably rename this to `DEFAULT_DATA_ATTRIBUTE` to match the terminology used (ie, `plotlyDataAttribute`).